index.html

```diff
@@ -609,106 +609,222 @@
     This section shows how developers can make use of the various features of
     this specification.
   </p>
-  <pre title="Push a text string to a peer device" class="example highlight">
-    // Specifying an option for pushing to peers only,
-    // and thus, do nothing when the user taps a tag.
-    navigator.nfc.push("Text meant for peers only", { target: "peer" })
-      .then(() => {
-        console.log("Message pushed.");
-      }).catch((error) => {
-        console.log("Push failed :-( try again.");
-      });
-  </pre>
-  <pre title="Push a text string to either a tag or peer"
-       class="example highlight">
-    // Push a text string (here: serialized JSON), with no options specified
-    // (target defaults to any device).
-    navigator.nfc.push('{ prop1: "value1"; prop2: "value2" }')
-      .then(() => {
-        console.log("Message pushed.");
-      }).catch((error) => {
-        console.log("Push failed :-( try again.");
-      });
-  </pre>
-  <pre title="Push a URL to either a tag or peer" class="example highlight">
-    // To push an NDEF record of URL type, use NFCMessage
+
+  <aside title="Push a text string to either a tag or peer"
+       class="example">
+    <p>
+      Pushing a text string (like the serialized JSON below)
+      to any kind of device is straight forward. Options can
+      be left out, as they default to pushing to both tags and
+      peers.
+    </p>
+    <pre class="highlight">
+navigator.nfc.push(
+    '{ "prop1": "value1"; "prop2": "value2" }'
```

Comment:
should be comma
'{ "prop1": "value1", "prop2": "value2" }'

x = JSON.parse('{ "prop1": "value1"; "prop2": "value2" }'); => exception